static/gsApp/hooks/spendVisibility/enhancedUsageStatsOrganization.tsx

```diff
@@ -52,354 +51,261 @@ import {
  * this is a limitation to our app's ability to display spike history.
  */
 const REQUIRED_INTERVAL = '1h';
-/**
- * We allow 403s since we don't know if spike protection is enabled for the project or not
- * If not, don't raise an error since we can still show usage data.
- * We allow 404s in case they don't have access to new spike protection just yet
- */
-const ALLOWED_STATUSES = new Set([403, 404]);
```

Comment:
Now we are using `useApiQuery` and will not block the whole page if an endpoint fails. I believe this no longer makes sense